### DISCUSSION TITLE
How does FunctionParameter() work

### DISCUSSION THREAD

**Discussion by loveinSH** [pybamm-team/PyBaMM#3599] [Q&A]
Hello, when I look into the code follows, I found that we can not find the value of Exchange-current density for stripping directly. Neither in code lithium_ion_parameters nur in code plating. I want to know where did PyBaMM give the definition and the equation of Exchange-current density for stripping or how does FunctionParameter() works.
Thank you for the reply.
def j0_stripping_dimensional(self, c_e, c_Li, T):
        """Dimensional exchange-current density for stripping [A.m-2]"""
        inputs = {
            "Electrolyte concentration [mol.m-3]": c_e,
            "Plated lithium concentration [mol.m-3]": c_Li,
            "Temperature [K]": T,
        }
        return pybamm.FunctionParameter(
            "Exchange-current density for stripping [A.m-2]", inputs
        )

**Comment by valentinsulzer** [comment]
it's in whichever parameter file is being used (e.g. if using ParameterValues("Chen2020") it's in `Chen2020.py`)
